src/styles/button.scss

```diff
@@ -0,0 +1,24 @@
+.to-buy {
+  display: block;
+  width: 166px;
+  margin-left: $padding-left;
+  border: 1px solid $blue_color;
+  margin-top: 16px;
+  background-color: $blue_color;
+  color: $white_color;
+  text-align: center;
+  line-height: 40px;
+  height: 40px;
+  border-radius: $border-radius;
+  font-weight: 700;
+  font-size: 14px;
+  font-family: $font-family;
```

Comment:
The variable `$font-family` is used but not defined. Please define this variable to avoid compilation errors.